creator-node/src/services/stateMachineManager/index.js

```diff
@@ -64,6 +28,8 @@ class StateMachineManager {
       )
       logger.error(`updateEndpointToSpIdMap Error: ${e.message}`)
     }
+
+    await this.stateMonitoringQueue.init(audiusLibs)
```

Comment:
this partially addresses my [earlier coment](https://github.com/AudiusProject/audius-protocol/pull/3142#discussion_r880941245)
whats benefit of calling `updateEnabledReconfigModesSet()` in constructor?
come to think of it, why even call `new StateMachineMonitoringQueue()`
would be nice to only look in one place for everything right - leave constructor fully empty and put all in `init()`?

Reply:
same question applies to `stateMonitoringQueue.constructor` as well, curious what you think here

(can handle after merge)

Reply:
I copied over calling it in the constructor because snapback calls it there to update the internal class variables. as far as it being in constructor vs init, I agree I think it'll be cleaner to do everything in init and not use the constructor. making that change in a followup PR